internal/devbox/providers/identity/identity.go

```diff
@@ -4,12 +4,12 @@ import (
 	"context"
 	"os"
 
+	"go.jetify.com/typeid"
 	"go.jetpack.io/devbox/internal/build"
 	"go.jetpack.io/pkg/api"
 	"go.jetpack.io/pkg/auth"
 	"go.jetpack.io/pkg/auth/session"
 	"go.jetpack.io/pkg/ids"
-	"go.jetpack.io/typeid"
 	"golang.org/x/oauth2"
```

Comment:
Not for this PR, but when should we just update all these dependencies to use `go.jetify.com` instead?